microk8s-resources/wrappers/microk8s-disable.wrapper

```diff
@@ -1,76 +1,14 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+set -eu
 
 export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
 ARCH="$($SNAP/bin/uname -m)"
-export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
-export LD_LIBRARY_PATH=$SNAP_LIBRARY_PATH:$LD_LIBRARY_PATH
+export IN_SNAP_LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
+export PYTHONNOUSERSITE=false
 
 source $SNAP/actions/common/utils.sh
 
-if [ -e ${SNAP_DATA}/var/lock/clustered.lock ]
-then
-  echo "This MicroK8s deployment is acting as a node in a cluster. Please use the microk8s disable on the master."
-  exit 0
-fi
-
-if echo "$*" | grep -q -- '--help'; then
-    prog=$(basename -s.wrapper "$0" | tr - .)
-    echo "Usage: $prog ADDON..."
-    echo "Disable one or more ADDON included with microk8s"
-    echo "Example: $prog dns storage"
-    echo
-    echo "Available addons:"
-    echo
-    actions="$(find "${SNAP}/actions" -name '*.yaml' -or -name 'disable.*.sh')"
-    actions="$(echo "$actions" | sed -e 's/.*[/.]\([^.]*\)\..*/\1/' | sort | uniq)"
-    for action in $actions; do
-        echo "  $action"
-    done
-    exit 0
-fi
-
-exit_if_stopped
```

Comment:
I think we should keep this check. If microk8s is stopped we do not want to try to enable anything, right?